docs/conf.py

```diff
@@ -168,33 +158,54 @@
 # variables set in the global configuration. The variables set in the
 # global configuration are listed below, commented out.
 
+html_css_files = ["jdaviz.css"]
+html_copy_source = False
+
+html_theme_options.update(  # noqa: F405
+    {
+        "github_url": "https://github.com/spacetelescope/jdaviz",
+        "external_links": [
+            {"name": "Help Desk", "url": "http://jwsthelp.stsci.edu/"},
+        ],
+        "use_edit_page_button": True,
+    }
+)
+
+html_context = {
+    "default_mode": "light",
+    "to_be_indexed": ["stable", "latest"],
+    "is_development": dev,
+    "github_user": "spacetelescope",
+    "github_repo": "jdaviz",
+    "github_version": "main",
+    "doc_path": "docs",
+}
 
 # Add any paths that contain custom themes here, relative to this directory.
 # To use a different custom theme, add the directory containing the theme.
-#html_theme_path = []
+# html_theme_path = []
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes. To override the custom theme, set this to the
 # name of a builtin theme or the name of a custom theme in html_theme_path.
-#html_theme = None
 
-html_theme = "sphinx_rtd_theme"
+# html_theme = "sphinx_rtd_theme"
 
 # Custom sidebar templates, maps document names to template names.
-#html_sidebars = {}
+# html_sidebars = {}
 
 # The name of an image file (relative to this directory) to place at the top
 # of the sidebar.
-#html_logo = ''
+html_logo = 'logos/jdaviz.svg'
 
 # The name of an image file (within the static path) to use as favicon of the
 # docs.  This file should be a Windows icon file (.ico) being 16x16 or 32x32
 # pixels large.
-#html_favicon = ''
+html_favicon = 'logos/specviz2d.ico'
```

Comment:
That is not a question I can answer. I could not find one in the current logo stash.

Reply:
(but doesn't need to block this PR, using specviz2d logo is still more appropriate than the astropy logo that used to be there - although I am seeing a white instead of transparent background which would also be nice to fix)

Reply:
what about the icon thats currently on the left hand side of the top menu, the one that says 'jdaviz' and has the 4 little icons under it (why not 5, by the way?)

Reply:
Keep in mind that this icon goes on the *browser tab*. There is very little space, so a full Jdaviz logo would be illegible.

Reply:
> why not 5, by the way?

I think the 5th is a combination of the other two? 🤷